modules/installation-gcp-permissions.adoc

```diff
@@ -32,6 +32,13 @@ account requires the following permissions. If you deploy your cluster into an e
 .Required roles for creating network resources during installation
 * DNS Administrator
 
+.Required roles for installing into a shared VPC
+* Compute Instance Admin (v1)
+* Compute Load Balancer Admin
+* Compute Storage Admin
+* IAM Role Viewer
```

Comment:
I couldn't find this in the preview. Would you mind providing a link to it?

Reply:
Oops, here you go: https://51171--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-account.html#installation-gcp-permissions_installing-gcp-account